tools/topology/topology2/include/components/google-rtc-aec.conf

```diff
@@ -138,4 +143,6 @@ Class.Widget."google-rtc-aec" {
 	is_pages		1
         num_input_pins          2
         num_output_pins         1
+        # AEC requires 10ms buffers for processing
+        buffer_period_size	10
```

Comment:
Should this be buffer_period_ms ?

Reply:
I can use BUFFER_PERIOD_MS in the 2nd version.